### PR TITLE
FOSSC-86 jquery

### PR DESCRIPTION
**Pull request by dierberg**

### CODE REVIEW

File: analysed-packages/jquery/version-3.4.1/jquery-3.4.1-SPDX2TV.spdx [hunk 1]
```diff
@@ -0,0 +1,4203 @@
...
+FileCopyrightText: <text> Copyright(c) 2013 jake luer <jake@alogicalparadox.com> MIT Licensed </text>
+
+##File
+
+FileName: jquery-3.4.1.tar.gz/jquery-3.4.1.tar/jquery-3.4.1/external/requirejs/require.js
+SPDXID: SPDXRef-item491702
+FileChecksum: SHA1: 33bc97e78f4905e72b4c1eb2ca0a4662588443e3
+FileChecksum: SHA256: 9485f0917f97fcf4f63a5ea365200ffd57f123f451382a2f9a1ad2e2fd51ac9b
+FileChecksum: MD5: f0cc8bbb2fcef87fc194fecbb632fcfa
+LicenseConcluded: LicenseRef-MIT AND LicenseRef-RequireJS
```
Comment (OliverFendt) [inline analysed-packages/jquery/version-3.4.1/jquery-3.4.1-SPDX2TV.spdx:118]: Why is there LicenseRef-MIT AND LicenseRef-RequireJS cocnluded as far as I can see there is only the statement 

> Released under MIT license, https://github.com/requirejs/requirejs/blob/master/LICENSE

this means in my opinion this file is licensed under MIT and you can find the license text at the given link - which you have than concluded to RequireJS, which is correct but the MIT conclusion is to much here

File: analysed-packages/jquery/version-3.4.1/jquery-3.4.1-OSS-disclosure.txt [hunk 2]
```diff
@@ -0,0 +1,1620 @@
...
+
+Files located in the node_modules directory, and certain utilities used
+to build or test the software in the test and dist directories, are
+externally maintained libraries used by this software which have their own
+licenses; we recommend you read them, as their terms may differ from the
+terms above.
+
+--------------------------------------------------------------------------------
+
+warranty-disclaimer
```
Comment (OliverFendt) [inline analysed-packages/jquery/version-3.4.1/jquery-3.4.1-OSS-disclosure.txt:102]: Please see all my comments in the tag value file

File: analysed-packages/jquery/version-3.4.1/jquery-3.4.1-SPDX2TV.spdx [hunk 3]
```diff
@@ -0,0 +1,4203 @@
...
+PackageChecksum: MD5: 35184010253a13f3512b7995a7a4d51d
+PackageLicenseConcluded: NOASSERTION
+PackageLicenseDeclared: LicenseRef-MIT
+PackageLicenseComments: <text> licenseInfoInFile determined by Scanners:
+ - nomos ("4.4.0".903df0)
+ - monk ("4.4.0".903df0)
+ - ojo ("4.4.0".903df0)
+ - scancode ("4.4.0".903df0) </text>
+PackageLicenseInfoFromFiles: NOASSERTION
+PackageCopyrightText: NOASSERTION
```
Comment (OliverFendt) [inline analysed-packages/jquery/version-3.4.1/jquery-3.4.1-SPDX2TV.spdx:48]: In the disclosure file you have listed the contributors but not here, why? can you please add them here too:

> PackageCopyrightText: <text> Authors ordered by first contribution. .... </text>

File: analysed-packages/jquery/version-3.4.1/jquery-3.4.1.spdx.json
Comment (OliverFendt) [inline analysed-packages/jquery/version-3.4.1/jquery-3.4.1.spdx.json:1]: Please resolve in this file my comments in the disclosure file and tag value file, too

File: analysed-packages/jquery/version-3.4.1/jquery-3.4.1.spdx.rdf.xml
Comment (OliverFendt) [inline analysed-packages/jquery/version-3.4.1/jquery-3.4.1.spdx.rdf.xml:1]: Please resolve in this file my comments in the disclosure file and tag value file, too

File: analysed-packages/jquery/version-3.4.1/jquery-3.4.1.spdx.yaml
Comment (OliverFendt) [inline analysed-packages/jquery/version-3.4.1/jquery-3.4.1.spdx.yaml:1]: Please resolve in this file my comments in the disclosure file and tag value file, too

**Review by OliverFendt — CHANGES_REQUESTED** [review]
Awesome work @dierberg , I only have some minor comments

**Comment by dierberg** [comment]
new documents generated

**Review by OliverFendt — APPROVED** [review]
Great, thank you very much for your effort

**Pull request merged**